pkg/vm/engine/tae/containers/runtime_1.22.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build go1.22
+
+package containers
+
+import (
+	_ "unsafe"
+)
+
+//go:linkname fastrand runtime.cheaprand
+func fastrand() uint32
```

Comment:
Do you want a cheap rand here?  I am not sure how this works but I don't see any difference in 1.22 from older.

Reply:
> Do you want a cheap rand here? I am not sure how this works but I don't see any difference in 1.22 from older.

In Go 1.22, to ensure compatibility, `runtime.fastrand` is effectively an alias for `runtime.rand`.

In Go 1.21, `runtime.fastrand` can be found here: https://github.com/golang/go/blob/go1.21.0/src/runtime/stubs.go#L124-L148. However, in Go 1.22, it has been renamed to `runtime.cheaprand`, as shown here: https://github.com/golang/go/blob/go1.22.0/src/runtime/rand.go#L178-L202.

In Go 1.22, `runtime.fastrand` serves as an alias for `runtime.legacy_fastrand`, which in turn calls `runtime.rand`.

There is a noticeable performance difference between `runtime.rand` and `runtime.cheaprand` in Go 1.22, which may require attention in performance-sensitive scenarios.

Additionally, references to `runtime.fastrand` are set to be removed in Go 1.24.